internal/integrate/integrator.go

```diff
@@ -745,25 +532,14 @@ func writeRuleToFile(rule *definitions.ProvisionedAlertRule, outputFile string,
 	return nil
 }
 
-func escapeQueryJSON(query string) (string, error) {
-	escapedQuotedQuery, err := json.Marshal(query)
-	if err != nil {
-		return "", fmt.Errorf("could not escape provided query: %s", query)
-	}
-	return string(escapedQuotedQuery[1 : len(escapedQuotedQuery)-1]), nil // strip the leading and trailing quotation marks
-}
-
+// getC is a helper function that returns the first non-empty value
+//
+// Deprecated: Use shared.GetConfigValue instead
```

Comment:
Are we keeping this just to reduce the amount of changes needed? We're switching to `shared` in a lot of other places. We could just include that here, or do a follow-up. Definitely not a blocker!